### ISSUE TITLE
Glibc update breaks Asterisk

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#6420]
**Steps to reproduce**

- update glibc to version glibc-2.17-322
- restart Asterisk
- try to make a call

**Expected behavior**

- Call should work as before

**Actual behavior**

- call doesn't work, "call-limit-reached" is played
- on `/var/log/asterisk/full` logs, if are evaluated wrong ("nan" should not be there)
```
[2021-02-05 15:51:05] VERBOSE[22786][C-00000002] pbx.c: Executing [s@macro-user-callerid:6] GotoIf("PJSIP/201-00000002", "nan?limit") in new stack
```

**Components**
asterisk13-*-13.38.1-1.ns7.x86_64
glibc-common-2.17-322.el7_9.x86_64
glibc-2.17-322.el7_9.x86_64

**See also**

https://community.nethserver.org/t/asterisk-is-offline/17582

EDIT: upstream bug https://bugzilla.redhat.com/show_bug.cgi?id=1925204

----

Thanks to PCXLan.es that reported the issue

**Comment by ibinetwork** [comment]
Update fix glibc-2.17-323.el7_9.x86_64 released

**Comment by Stell0** [comment]
glibc-2.17-323.el7_9.x86_64 fix the issues
